fedimint-api-client/src/api/global_api/with_cache.rs

```diff
@@ -364,13 +364,15 @@ where
         &self,
         name: String,
         federation_name: Option<String>,
+        disable_base_fees: Option<bool>,
```

Comment:
Would generally avoid negatives, a go for `enable_`. The meaning of `None` vs `Some` is not immediately clear too.

Reply:
I get it now and it makes *some* sense. Instead of being an override the env var turning on/off base fees is only taken into account if the leader didn't set that flag. Having clear precedence of the env var would be easier imo.